content/advent-2018/llvm-ir-and-go.md

```diff
@@ -0,0 +1,311 @@
++++
+title = "LLVM IR and Go"
+date = 2017-12-19T08:00:00Z
+author = ["Robin Eklind"]
+series = ["Advent 2018"]
++++
+
+<!-- TODO: remember to update date to 2018 -->
+
+In this post, we'll look at how to build Go programs -- such as compilers and static analysis tools -- that interact with the LLVM compiler framework using the LLVM IR assembly language.
+
+_**TL;DR** we wrote a library for interacting with LLVM IR in pure Go, see links to [code](https://github.com/llir/llvm) and [example projects](https://github.com/llir/llvm#users)._
+
+<!-- TODO: remove 3. Building a toy compiler in Go section? If time allows, add it :) -->
+<!-- 3. Building a toy compiler in Go -->
+
+1. [Quick primer on LLVM IR](#quick-primer-on-llvm-ir)
+2. [LLVM IR library in pure Go](#llvm-ir-library-in-pure-go)
+3. [Closing notes](#closing-notes)
+4. [Further resources](#further-resources)
+
+## Quick primer on LLVM IR
+
+_(For those already familiar with LLVM IR, feel free to [skip this section](#llvm-ir-library-in-pure-go))._
```

Comment:
`s/skip this section/jump to this section/` ?

Reply:
Thanks, sound better.